nullaway/src/main/java/com/uber/nullaway/AbstractConfig.java

```diff
@@ -237,6 +241,18 @@ public boolean isExternalInitClassAnnotation(String annotationName) {
     return externalInitAnnotations.contains(annotationName);
   }
 
+  @Override
+  @Nullable
+  public String getContractString(Symbol.MethodSymbol methodSymbol) {
```

Comment:
I am a bit concerned about making Checker Framework javacutils in the configuration classes, since those mostly should be to parse out CLI argument information and leave the actual code checking to other parts of NullAway. 

Would it be possible to refactor things so that this method lives in e.g. `com.uber.nullaway.handlers.contract.ContractUtils` (as a `static` method) and `Config` merely provides a:

```
public Set<String> getConstractAnnotations() {
    ...
}
```

method?

That way, the places were you have:

```
config.getContractString(callee)
```

Become:

```
ContractUtils.getContractString(config.getConstractAnnotations())
```

Which is slightly more verbose, but preserves the layering between configuration classes and handlers.